code/public/index.html

```diff
@@ -30,5 +53,19 @@
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
   </body>
-
+  <footer>
+    <div class="footer-wrapper">
+      <a
+        href="https://davidballester.dev"
+        target="_blank"
+      >
+        <p>© David Ballester</p>
+      </a>
+      <a
+        href="https://icons8.com/"
+        target="_blank"
+        ><p>assets source</p></a
+      >
+    </div>
+  </footer>
```

Comment:
Maybe this could have been placed in app.js instead to follow the same structure as for inner- and outer-section :)